### PR TITLE
Alphabetize and Format ScriptLib.java

### PR DESCRIPTION
**Pull request by scooterboo**
## Description
I deeply apologize to anyone who is trying to find the history of the code from before today.

- [ ] Bug fix
- [ ] New feature 
- [ ] Enhancement
- [x] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

**Comment by Hartie95** [comment]
Might be more of a preferences things, but isn't it better in the long run to have them grouped by their function? 
Otherwise you will have for example all variable handling on completely different parts of the file, making it hard to follow usage.

**Comment by scooterboo** [comment]
I tried doing that but gave up on it. There's just too many functions.

There would be trouble where it would be hard for new people to follow a sorting that like. If a new function was made, it'd be very likely that someone would insert it into the wrong place.

Better would be if we somehow could manage to split scriptlib into multiple files.

**Comment by eternalcomet** [comment]
Modern IDE like vscode and intellij idea are able to show the structure or outline of the file, which make it easier to find specific method. Besides, as you mentioned above, such refactor make it almost impossible to find code history.

**Review by KingRainbow44 — APPROVED** [review]

**Comment by KingRainbow44** [comment]
> such refactor make it almost impossible to find code history

intellij idea's git viewer allows you to see the commit history of a file, including who changed it. this shouldn't affect anything.
